ecommerce/src/main/kotlin/io/hhplus/ecommerce/config/p6spy/P6spyPrettySqlFormatter.kt

```
@@ -0,0 +1,118 @@
+//    }
+//
+//    /**
+//     * DML(Data Manipulation Language) 쿼리인지 확인
+//     */
+//    private fun isDmlStatement(sql: String): Boolean {
+//        return sql.startsWith("select") ||
+//                sql.startsWith("insert") ||
+//                sql.startsWith("update") ||
+//                sql.startsWith("delete")
+//    }
+//}
```

detekt (1.23.8)

[warning] The file does not contain a package declaration.

(detekt.potential-bugs.MissingPackageDeclaration)